src/main/scala/strawman/collection/Map.scala

```diff
@@ -214,6 +214,8 @@ trait MapOps[K, +V, +CC[X, Y] <: MapOps[X, Y, CC, _], +C <: MapOps[K, V, CC, C]]
 
   override def hashCode(): Int = Set.unorderedHash(toIterable, "Map".##)
 
+  override def toString(): String = super[IterableOps].toString()
```

Comment:
At the same time I fixed `toString` of `Map` collections. Previously, the `toString` implementation of `PartialFunction` was chosen.